lib/sampler.js

```diff
@@ -200,8 +189,6 @@ module.exports = {
     if (process.cpuUsage) {
```

Comment:
I think you can remove this too. all versions of node.js will have `process.cpuUsage` that we support. there was a use case that you could be in < 6.1.0 and not have native metrics installed as it was optional but none of that is an issue now

Reply:
Ah, of course. Will do.